subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,80 @@
+
+#### Comprehension
+
+##### Ask the student to describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
+
+##### Ask the student to explain one of the Ansible playbook
+
+###### did he/she clearly explain all the Ansible playbook
+
+##### Discuss the CI/CD pipeline setup.
+
+###### is there unit tests for each functionality and are the test run for each PR.
+
+###### is SonarQube report free from any error or warning that can break the CI/CD Process.
+
+##### Detail the security measures implemented.
+
+###### Were comprehensive security measures like SSL/TLS, secret management, and principle of least privilege correctly implemented?
+
+##### Ask the student to explain the database schema for PostgreSQL and Neo4j.
+
+###### Did the data structure in PostgreSQL and Neo4j effectively support the application's requirements?
+
+#### Functional
+
+##### Verify the execution of Ansible playbooks.
+
+###### Did the Ansible playbooks execute without errors and configure the environment as intended?
+
+###### Were the playbooks able to handle re-running scenarios without causing disruptions or inconsistencies?
+
+##### Test each microservice API.
+
+###### Are all microservice APIs is role based and doesn't work with a admin role based user authenticated except the login api?
+
+###### Are all microservice APIs function correctly and handle errors effectively?
+
+##### Verify Admin Dashboard functionality.
+
+###### Did the Admin Dashboard correctly manage users, travels, and payment gateways?
+
+##### Test Authentication and Authorization.
+
+###### Was the authentication service robust and did the role-based access control function correctly?
+
+##### Ask the student to Simulate load on microservices.
```

Comment:
`... for example, by introducing some infinite loops in their API handlers.`

Or any other way you see might help to simulate extra load in the microservices

Reply:
for this part is up to the students they should use there own way to prove that to the auditor